README.md

```diff
@@ -154,6 +154,12 @@ public void some_important_test() throws Exception {
 }
 ```
 
+## Magic that Barista does for you
+
+In order to speed up testing, Barista keeps in mind some considerations.
+- **Scrolls when needed**: Filling a form inside a `ScrollView` requests you to scroll to the next item several times to avoid interacting with undisplayed widgets. To keep tests concise, Barista scrolls automatically before clicking a `Button` and before writing into an `EditText`.
```

Comment:
> before clicking a Button

Actually, clicking anything. This sentence might lead someone to think that it only works for buttons. Suggestion: Before clicking any View.

Also, I think it doesn't say clear enough how shitty is scrolling in Espresso.Here's an idea:
_"Interacting with Espresso in a `ScrollView` requires you to scroll to each view, which sometimes doesn't work the first time. Also trying to scroll outside a ScrollView makes produces an Exception, forcing you to change the test depending on the layout. To keep tests simpler. Barista scrolls....."_

It's probably too much text. But scrolling in Espresso is **so frustrating** that I think a small sentence doesn't give Barista enough credit.